go/vt/vtctl/query.go

```diff
@@ -445,6 +445,10 @@ func (lw loggerWriter) Write(p []byte) (int, error) {
 
 // printQueryResult will pretty-print a QueryResult to the logger.
 func printQueryResult(writer io.Writer, qr *sqltypes.Result) {
+	if len(qr.Rows) == 0 {
```

Comment:
can qr be nil here?

Reply:
good observation. Actually, it should never be `nil`, and yet it's good to protect against such a case -- fixed.